### PR TITLE
Html&CSS Popup

### PR DESCRIPTION
**Pull request by strumerlz**
### HTML & CSS practice: Hooli-style Popup

[Demo](https://strumerlz.github.io/PopUpTask) 
[Code base](https://github.com/strumerlz/kottans-frontend/tree/main/HTML-CSS-Popup)

The code is submitted in a dedicated feature branch.
Please, review.

### CODE REVIEW

File: submissions/strumerlz/html-css-popup/index.html [hunk 1]
```diff
@@ -0,0 +1,160 @@
+<!DOCTYPE html>
+<html lang="en">
+
+    <head>
+        <meta charset="utf-8">
+        <title>HTML & CSS practice: Pop-Up Menu</title>
+        <link rel="preconnect" href="https://fonts.gstatic.com">
```
Comment (lenadgit) [inline submissions/strumerlz/html-css-popup/index.html:7]: Metatag Viewport is mandatory for mobile markup.  
https://serpstat.com/ru/blog/chto-takoe-metateg-viewport/
Could you please add it?
`<meta name="viewport" content="width=device-width, initial-scale=1.0">`

**Review by lenadgit — CHANGES_REQUESTED** [review]

File: submissions/strumerlz/html-css-popup/index.html [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+                                    </a>
+                                    </li>
+                                    <li class="submenu-item">
+                                    <a href="#">
+                                        <img class="submenu-icon" src="assets/submenu_icons/idea.png" alt="Ideas">
+                                        <span>Ideas</span>
+                                    </a>
+                                    </li>
+                                </ul>
+
```
Comment (lenadgit) [inline submissions/strumerlz/html-css-popup/index.html:86]: could you please remove spaces (empty lines)? would be better to use code formatting (looks like Prettier, etc, it depends on what you use)

**Review by lenadgit — CHANGES_REQUESTED** [review]

File: submissions/strumerlz/html-css-popup/style.css [hunk 3]
```diff
@@ -0,0 +1,225 @@
...
+
+* {
+  margin: 0;
+  padding: 0;
+}
+
+html{
+    font-family: 'Poppins', sans-serif;
+    color: var(--basic-color);
+
```
Comment (lenadgit) [inline submissions/strumerlz/html-css-popup/style.css:15]: could you please format your file by any auto-formatting? (it removes empty lines, adds necessary alignments)

**Review by lenadgit — CHANGES_REQUESTED** [review]

**Comment by lenadgit** [comment]
Nice job! Simply and creatively

File: submissions/strumerlz/html-css-popup/style.css [hunk 4]
```diff
@@ -0,0 +1,225 @@
...
+     text-align: end;
+     padding-right: 10px;
+     line-height: 3rem;
+     
+ }
+ footer a {
+    color: inherit;  
+ }
+ 
+
```
Comment (lenadgit) [inline submissions/strumerlz/html-css-popup/style.css:225]: if you see this symbol on github at the end of the file, it means that empty line is needed at the end.
empty line
https://github.com/kottans/frontend-2021-homeworks/pull/229/files
Could you please add an empty line but without any spaces (you added two new lines but with space, Github reads this as a symbol))

**Review by lenadgit — CHANGES_REQUESTED** [review]

File: submissions/strumerlz/html-css-popup/index.html (hunk 1 above)
Comment (strumerlz) [reply submissions/strumerlz/html-css-popup/index.html]: Fixed it.

File: submissions/strumerlz/html-css-popup/index.html (hunk 2 above)
Comment (strumerlz) [reply submissions/strumerlz/html-css-popup/index.html]: Thanks for advice! Corrected it.

File: submissions/strumerlz/html-css-popup/style.css (hunk 3 above)
Comment (strumerlz) [reply submissions/strumerlz/html-css-popup/style.css]: Corrected it.

File: submissions/strumerlz/html-css-popup/style.css [hunk 5]
```diff
@@ -0,0 +1,225 @@
...
+.horizontal-rule {
+    display: none;
+    border: 1px solid silver;
+    margin: 0.25rem 0.5rem 0.25rem 0.5rem;
+}
+
+.submenu-more-label:hover,
+.kottans-link:hover, 
+.kottans-link:focus,
+#more-hidden-checkbox:focus ~ .submenu-more-label {
```
[the remark this answers is not in the document]
Comment (strumerlz) [reply submissions/strumerlz/html-css-popup/style.css]: Thank you, replaced all ID for classes. I'll know better next time.

File: submissions/strumerlz/html-css-popup/style.css (hunk 4 above)
Comment (strumerlz) [reply submissions/strumerlz/html-css-popup/style.css]: Oh, thanks, didn't see those spaces. Corrected it.

**Review by lenadgit — APPROVED** [review]
Thank you for the submission. Feel free taking the next course module while you submission is being reviewed

**Pull request merged**